html/semantics/popovers/popover-attribute-all-elements.html

```diff
@@ -33,7 +33,23 @@
       assertNotAPopover(element);
     }, `A <${tag}> element should *not* behave as a popover.`);
   });
-
+  elementsThatDontRender.forEach(tag => {
+    test((t) => {
+      const element = document.createElement(tag);
+      element.setAttribute('popover','auto');
+      document.body.appendChild(element);
+      t.add_cleanup(() => element.remove());
+      assertPopoverVisibility(element, false, false, 'should not behave as a popover');
+    }, `A <${tag} popover> element should *not* behave as a popover.`);
+    test((t) => {
+      const element = document.createElement(tag);
+      document.body.appendChild(element);
+      t.add_cleanup(() => element.remove());
+      assert_throws_dom("NotSupportedError",() => element.showPopover(),'Calling showPopover on a non-popover should throw NotSupported');
+      assert_throws_dom("NotSupportedError",() => element.hidePopover(),'Calling hidePopover on a non-popover should throw NotSupported');
+      assert_throws_dom("NotSupportedError",() => element.togglePopover(),'Calling togglePopover on a non-popover should throw NotSupported');
```

Comment:
Ditto here, there is no such restriction in the HTML spec.

Reply:
Checking for display instead is not working in certain cases:
FAIL A <br> element should *not* behave as a popover. assert_equals: Element isn't rendered expected "none" but got "inline"
FAIL A <slot> element should *not* behave as a popover. assert_equals: Element isn't rendered expected "none" but got "contents"
etc.
Not sure what to test here so I removed this section.